pkg/graph/schema.graphql

```diff
@@ -53,6 +53,35 @@ type CedarSystem {
 	systemMaintainerOrgComp: String
 }
 
+"""
+"""
+type CedarSystemBookmark {
+  euaUserId: String!
+  cedarSystemId: UUID!
+}
+
+"""
+The data needed to bookmark a cedar system
+"""
+input CreateCedarSystemBookmarkInput {
+  cedarSystemId: UUID!
```

Comment:
I don't think cedar system IDs should be marked as UUIDs - right now in the cedar system they look something like `"326-1556-0"` - let's just make them strings for now!

Reply:
d4f1c3110a12d0f18388d54c470850d74ba7abc3